app/services/bus_logic/position_checker.py

```
@@ -0,0 +1,147 @@
+def get_stations_by_position(service_key, tm_x, tm_y, radius=500):
+    """좌표 기준 반경 내 정류소 조회"""
+    url = 'http://ws.bus.go.kr/api/rest/stationinfo/getStationByPos'
+    params = {
+        'serviceKey': service_key, 
+        'tmX': tm_x, 
+        'tmY': tm_y, 
+        'radius': radius
+    }
+    
+    try:
+        response = requests.get(url, params=params)
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)